esp/logging/loggingagent/cassandraloggingagent/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ if(USE_CASSANDRA)
   ADD_DEFINITIONS( -D_USRDLL -DCASSANDRALOGAGENT_EXPORTS )
 
   set ( SRCS
+    ${HPCC_SOURCE_DIR}/esp/logging/datafieldmap.cpp
```

Comment:
I'm not a fan in general of including the same source files in multiple dlls (apart from anything else you end up with confusion of which name gets resolved...)

Is there a reason this functionality is not in a shared dll?